openlibrary/i18n/pl/messages.po

```diff
@@ -0,0 +1,3607 @@
+"zamiast podkreślenia tekstu) umieść ukośnik \\ przed tym znakiem."
+
+#: nav_foot.html:13
+msgid "Vision"
+msgstr "Wizja"
+
+#: nav_foot.html:14
+msgid "Jobs"
+msgstr "Oferty pracy"
+
+#: nav_foot.html:14
+msgid "Careers"
+msgstr "Kariera"
+
+#: nav_foot.html:15
+msgid "Blog"
+msgstr "Blog"
+
+#: nav_foot.html:16
+msgid "Terms of Service"
+msgstr "Warunki świadczenia usług"
+
+#: nav_foot.html:17
+msgid "Donate"
+msgstr "Dorzuć się"
+
+#: nav_foot.html:21
+msgid "Discover"
+msgstr "Odkrywaj"
+
+#: nav_foot.html:23
+msgid "Go home"
+msgstr "Wróć do strony głównej"
+
+#: nav_foot.html:23
+msgid "Home"
+msgstr "Strona główna"
+
+#: nav_foot.html:24
+msgid "Explore Books"
+msgstr "Zobacz książki"
+
+#: nav_foot.html:24
+msgid "Books"
+msgstr "Książki"
+
+#: nav_foot.html:25
+msgid "Explore authors"
+msgstr "Zobacz autorów"
+
+#: authors.html:81 nav_foot.html:25 view.html:135
+msgid "Authors"
+msgstr "Autorzy"
+
+#: nav_foot.html:26
+msgid "Explore subjects"
+msgstr "Zobacz książki"
+
+#: advancedsearch.html:7 advancedsearch.html:10 advancedsearch.html:15
+#: nav_foot.html:27 nav_head.html:19 nav_head.html:101
+msgid "Advanced Search"
+msgstr "Wyszukiwanie zaawansowane"
+
+#: nav_foot.html:28
+msgid "Navigate to top of this page"
+msgstr "Wróć na górę tej strony"
+
+#: nav_foot.html:28
+msgid "Return to Top"
+msgstr "Przewiń do góry"
+
+#: nav_foot.html:32
+msgid "Develop"
+msgstr "Rozwijaj"
+
+#: nav_foot.html:34
+msgid "Explore Open Library Development Center"
+msgstr "Poznaj Centrum Rozwoju Open Library"
+
+#: nav_foot.html:34
+msgid "Development Center"
+msgstr "Centrum Rozwoju"
+
+#: nav_foot.html:35
+msgid "Explore Open Library APIs"
+msgstr "Poznaj API Open Library"
+
+#: nav_foot.html:35
+msgid "API Documentation"
+msgstr "Dokumentacja API"
+
+#: nav_foot.html:36
+msgid "Bulk Open Library data"
+msgstr "Dane zbiorcze Open Library"
+
+#: nav_foot.html:36
+msgid "Bulk Data Dumps"
+msgstr "Zrzuty danych zbiorczych"
+
+#: nav_foot.html:37
+msgid "Write a bot"
+msgstr "Napisz bota"
+
+#: nav_foot.html:37
+msgid "Writing Bots"
+msgstr "Pisanie botów"
+
+#: nav_foot.html:38
+msgid "Add a new book to Open Library"
+msgstr "Dodaj nową książkę do Open Library"
+
+#: nav_foot.html:44
+msgid "Help Center"
+msgstr "Centrum Pomocy"
+
+#: nav_foot.html:45
+msgid "Problems"
+msgstr "Problemy"
+
+#: nav_foot.html:45
+msgid "Report A Problem"
+msgstr "Zgłoś problem"
+
+#: nav_foot.html:46
+msgid "Suggest Edits"
+msgstr "Zaproponuj zmiany"
+
+#: nav_foot.html:46
+msgid "Suggesting Edits"
+msgstr "Proponowanie zmian"
+
+#: nav_foot.html:58
+msgid ""
+"Open Library is an initiative of the <a href=\"//archive.org/\">Internet "
+"Archive</a>, a 501(c)(3) non-profit, building a digital library of "
+"Internet sites and other cultural artifacts in  digital form. Other <a "
+"href=\"//archive.org/projects/\">projects</a> include the <a "
+"href=\"//archive.org/web/\">Wayback Machine</a>, <a "
+"href=\"//archive.org/\">archive.org
```

Comment:
@tfmorris Add parameter name

Reply:
Github has apparently gotten confused about where the last 4-5 comments align since they're all associated with the same block of code, not the code they were originally associated with.